src/components/Label/index.js

```diff
@@ -11,6 +11,11 @@ const Label = glamorous.label(
     flexDirection: "column",
     boxSizing: "border-box",
     width: "100%",
+    ":after": {
+      content: "''",
```

Comment:
it's not clear to me what this css is needed for, please add a comment inline explaining what it's doing

Reply:
it's a clearfix to allow the label to clear the floated children

Reply:
durp ok no need for comment

Reply:
the "''" threw me